staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -225,7 +225,7 @@ type StatefulSetStatus struct {
 	// observedGeneration is the most recent generation observed for this StatefulSet. It corresponds to the
 	// StatefulSet's generation, which is updated on mutation by the API Server.
 	// +optional
-	ObservedGeneration *int64 `json:"observedGeneration,omitempty" protobuf:"varint,1,opt,name=observedGeneration"`
+	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"varint,1,opt,name=observedGeneration"`
```

Comment:
> Can we do this? Isn't it a backward incompatible change?
> @deads2k @smarterclayton

Isn't the `v1beta2` serialization net new in 1.8?  If it is, then the proto serialization won't be a problem.  Not sure how you'd roundtrip through it though.  The compatibility of that bit depends on the meaning of nil versus 0 in the API itself.